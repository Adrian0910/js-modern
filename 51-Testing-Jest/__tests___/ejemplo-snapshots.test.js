const cliente = {
    nombre: 'Adrian Ibarra',
    balance: 500,
    tipo: 'Premium'
};

describe('Testing al cliente', () => {
    test('Es Adrian Ibarra', () => {
        expect(cliente).toMatchSnapshot(); //toMatchSnapshot para validar todo el OBJ
    });
});




