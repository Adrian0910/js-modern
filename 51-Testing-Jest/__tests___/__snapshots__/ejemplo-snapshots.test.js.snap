// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Adrian Ibarra 1`] = `
Object {
  "balance": 500,
  "nombre": "Adrian Ibarra",
  "tipo": "Premium",
}
`;
